spec/ci_smoke_test_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+# rubocop:disable all
+
+require 'rails_helper'
```

Comment:
shall we remove this file as it's for the testing purpose and we know the CI is working now already @Goose97

Reply:
yes, I'll remove this